hilti/toolchain/src/compiler/codegen/codegen.cc

```diff
@@ -31,6 +36,18 @@ inline const DebugStream Compiler("compiler");
 
 namespace {
 
+/// Helper to check whether a give module looks like a builtin module.
+bool isBuiltin(const declaration::Module& m, std::string_view parseExtension, Context* context) {
+    const auto parse_plugin = plugin::registry().pluginForExtension(parseExtension);
+    assert(parse_plugin);
+
+    auto fileDir = rt::filesystem::path(m.meta().location().file()).parent_path();
+
+    auto paths = (*parse_plugin).get().library_paths(context);
```

Comment:
This still fails when running as part of a Zeek build where the library paths point to install locations, not the files we actually use during the compile.